tournaments/index.js

```diff
@@ -841,6 +852,29 @@ var commands = {
 		runautodq: function (tournament) {
 			tournament.runAutoDisqualify(this);
 		},
+		scout: 'setscouting',
+		scouting: 'setscouting',
+		setscout: 'setscouting',
+		setscouting: function (tournament, user, params, cmd) {
+			if (params.length < 1) {
+				if (tournament.scouting) {
+					return this.sendReply("This tournament allows scouting.");
+				} else {
+					return this.sendReply("This tournament disallows scouting.");
+				}
+			}
+
+			var option = params[0].toLowerCase();
+			if (option === 'on' || option === 'true' || option === 'allow' || option === 'allowed')  {
+				tournament.scouting = true;
+				this.privateModCommand("(The tournament was set to allow scouting by " + user.name + ")");
```

Comment:
That's one of things I'm not sure what to do about those (and a reason why I said it's work in progress). Probably it should be public, however.